### DISCUSSION TITLE
Automate/Schedule Backups

### DISCUSSION THREAD

**Discussion by ghost** [mateusz-bak/openreads#706] [Ideas]
**Is your feature request related to a problem? Please describe.**

There are currently no settings to automatically back up OpenReads data without manually triggering a backup. This can be tedious if frequent changes are made.

**Describe the solution you'd like**

I really like the way the open source authentication app Aegis implements backup functionality. Users are prompted for a location/directory to store their backups and can then either manually trigger a backup or enable a setting to create a backup every time a change is made. Users can also specify the amount of backups to store in the directory (older ones are deleted to save space).

This may be overkill for an app like OpenReads, but even an option to automate backups daily/weekly/etc. would be useful.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by 0-BlackSpectrum-0** [comment]
How about just like Showly app it'd automatically backup whenever I've updated anything ie live backup!?

**Comment by OctaveLarose** [comment]
This is an old request, but still a feature I personally care about a lot. I was thinking that while full automation would be ideal, if that's too annoying to implement or not a priority, a simpler, alternative solution may be to add a reminder system instead. 

If I could tell the app to remind me once a week/month/... that it's been X time since the last backup, through a popup window at startup with a button that redirects to the Backup settings, this would largely achieve a similar effect as a proper automatic backup feature. It doesn't *exactly* solve the problem, but I think it would be very useful.
